extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -47,160 +74,366 @@ Because of standardization of markup, support for multiple providers, and improv
 </script>
 ```
 
-Thus, notice that:
+### Local service
+
+The following properties are defined in this configuration:
+
+<table>
+  <tr>
+    <th>Property</th>
+    <th>Values</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>type</code></td>
+    <td>"remote" or "iframe"</td>
+    <td>Default is "remote". The <a href="#iframe-mode">"iframe" mode</a> allows for messaging to be communicated to a publisher-provided iframe, instead through CORS requests to publisher provided endpoints.</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>authorizationUrl</code></td>
+    <td>&lt;URL&gt;</td>
+    <td>The HTTPS URL for the <a href="#authorization-endpoint">Authorization Endpoint</a>.</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>pingbackUrl</code></td>
+    <td>&lt;URL&gt;</td>
+    <td>(Optional) The HTTPS URL for the <a href="#pingback-endpoint">Pingback Endpoint</a>.</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>pingbackAllEntitlements</code></td>
+    <td>&lt;boolean&gt;</td>
+    <td>(Optional) Whether to send entitlements from all services to the <a href="#pingback-endpoint">Pingback Endpoint</a> or not.</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>actions.login</code></td>
+    <td>&lt;URL&gt;</td>
+    <td>The HTTPS URL for the <a href="#login-page">Login page</a>.</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>actions.subscribe</code></td>
+    <td>&lt;URL&gt;</td>
+    <td>The HTTPS URL for the <a href="#subscribe-page">Subscribe page</a>.</td>
+  </tr>
+</table>
+
+_&lt;URL&gt;_ values specify HTTPS URLs with substitution variables. The substitution variables are covered in more detail in the [URL Variables][13] section below.
+
+Here’s an example of a "local" service configuration:
 
-1.  The product ID is "norcal_tribune.com:basic" (`"productID": "norcal_tribune.com:basic"`).
-2.  This document is currently locked (`"isAccessibleForFree": false`).
+```html
+<script type="application/json" id="amp-subscriptions">
+  {
+    "services": [
+      {
+        "authorizationUrl": "https://pub.com/amp-authorisation?rid=READER_ID&url=SOURCE_URL",
+        "pingbackUrl": "https://pub.com/amp-pingback?rid=READER_ID&url=SOURCE_URL",
+        "pingbackAllEntitlements": true,
+        "actions":{
+          "login": "https://pub.com/amp-login?rid=READER_ID&url=SOURCE_URL",
+          "subscribe": "https://pub.com/amp-subscribe?rid=READER_ID&url=SOURCE_URL"
+        }
+      },
+      ...
+    ]
+  }
+</script>
+```
 
-## Microdata markup
+#### URL variables
+
+When configuring the URLs for various endpoints, the Publisher can use substitution variables. The full list of these variables are defined in the [AMP Var Spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md). In addition, this spec adds a few subscriptions-specific variables such as `READER_ID` and `AUTHDATA`.
+
+Some of the most relevant variables are described in the table below:
```

Comment:
Some of the most? Where are the rest? How many are there total?

Reply:
Will re-word this to be more explicit.